src/app/com.handtrixxx.OpenCloud.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.handtrixxx.OpenCloud</id>
+  <icon type="stock">com.handtrixxx.OpenCloud</icon>
+  <name>OpenCloud Desktop</name>
+  <summary>Desktop client for OpenCloud file synchronization</summary>
+  <developer_name>OpenCloud community</developer_name>
+  <developer>
+    <name>Niklas Stephan</name>
+    <email>opencloud@handtrixxx.com</email>
+    <url>https://niklas-stephan.de</url>
+    <role>contributor</role>
+  </developer>
```

Comment:
this is still incorrect. You never addressed it. https://github.com/flathub/flathub/pull/7322#discussion_r2619549347

I feel like a broken record:

<img width="933" height="235" alt="image" src="https://github.com/user-attachments/assets/8d6dc17a-c9b0-48ab-8551-e7971e62e891" />